habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -96,11 +96,12 @@ def __init__(
             walk_data["stop_pose"]["joints"].reshape(-1),
             mn.Matrix4(walk_data["stop_pose"]["transform"]),
         )
-        self.draw_fps = draw_fps
         self.dist_per_step_size = (
             self.walk_motion.displacement[-1] / self.walk_motion.num_poses
         )
 
+        self.draw_fps = draw_fps
```

Comment:
I think its confusing to define draw_fps 2 ways - through lin_speed and through direct parameter. I would remove one of the two.

Reply:
Should I set some default value in the init function for draw_fps? What happens if for some reason, we don't call that funcion (for instance in sandbox)